front/components/vaults/ConnectorDataUpdatedModal.tsx

```diff
@@ -7,20 +7,34 @@ import {
   Page,
   SparklesIcon,
 } from "@dust-tt/sparkle";
-import type { LightWorkspaceType } from "@dust-tt/types";
+import type { ConnectorProvider, LightWorkspaceType } from "@dust-tt/types";
+import { REMOTE_DATABASE_CONNECTOR_PROVIDERS } from "@dust-tt/types";
 import Link from "next/link";
 
 type DataSourceViewSelectionModalProps = {
   isOpen: boolean;
   onClose: () => void;
   owner: LightWorkspaceType;
+  connectorProvider: ConnectorProvider;
 };
 
 export const ConnectorDataUpdatedModal = ({
   isOpen,
   onClose,
   owner,
+  connectorProvider,
 }: DataSourceViewSelectionModalProps) => {
+  const isRemoteDbProvider =
+    REMOTE_DATABASE_CONNECTOR_PROVIDERS.includes(connectorProvider);
+
+  const title = isRemoteDbProvider
+    ? "Metadata Sync in Progress"
+    : "Data Sync in Progress";
+
+  const contentMessage = isRemoteDbProvider
+    ? "Databases, schemas, and tables metadata are being synced (this process is usually quick)."
```

Comment:
_Feel free to ignore. 🙇🏻‍♀️_ 

From my understanding the goal of this modal is to explain we do not have the data right away after adding a new connection, since indeed the first setup of a connector can take several hours. 

On our case, since we do not sync the data and it's super quick, I don't think we should display the big red scary disclaimer. WDYT? 

I would probably keep the ContentMessage but not in "warning" variant and to clarify the one thing that is useful to know when we just set-up this type of connection.

Reply:
I agree. I think we want to make it non-scary, while still saying that some "sync" is happening but expected to be fast. The important stuff I think is that we tell you that you need to add some stuff to either company data or private vaults.

Reply:
Changed the color (see the new screenshot).

Wording is different than the usual one. Still want to mention that sync is in progress, as it could potentially take up to a couple minutes for a huge snowflake.